cmd/reviewdog/main.go

```diff
@@ -152,13 +154,26 @@ const (
 			$ export GERRIT_REVISION_ID=ed318bf9a3c
 			$ export GERRIT_BRANCH=master
 			$ export GERRIT_ADDRESS=http://localhost:8080
+	
+	"bitbucket-code-report"
```

Comment:
I'm not familiar with bitbucket and I don't have a strong opinion, but do you have any reason that it uses `bitbucket-code-report` instead of `bitbucket-code-insights`?

I'm actually either is fine, but it seems code insights is the official name, is that right?

Reply:
SG.